contrib/python/src/python/pants/contrib/python/checks/checker/3rdparty/BUILD

```diff
@@ -0,0 +1,4 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+python_requirements()
```

Comment:
Since all these deps are used in just one place, together, you could forgo the requirements.txt and the separate targets per requirement, and just have

```
python_requirement_library(
  requirements = [
    python_requirement('six>=1.9.0,<2')
    python_requirement('pycodestyle==2.4.0'),
    python_requirement('pyflakes==2.0.0'),
  ]
)
```

Then you can depend on all of them at once by depending on 

`contrib/python/src/python/pants/contrib/python/checks/checker/3rdparty`

Which would be more succinct.

Reply:
I wanted to use this approach, but the downside is that `virtual_env` and `release.sh` won't know to install / build wheels for those 3 dependencies, which we need them to do.